src/Files.App/Services/Settings/IGeneralSettingsService.cs

```diff
@@ -207,6 +207,6 @@ public interface IGeneralSettingsService : IBaseSettingsService, INotifyProperty
 		/// <summary>
 		/// A dictionary to determine the custom hotkeys
 		/// </summary>
-		Dictionary<string, string> Actions { get; set; }
+		Dictionary<string, string>? Actions { get; set; }
```

Comment:
This can be not null, or not? The implementation seems safe

Reply:
The default json string is like `Actions: null` but previously once initialize and empty it, it will be `Actions: {}`. Making it nullable makes sense in this case, I assume. There's a code line ActionsViewModel that makes it null to restore all default (if not nullable, a warning will persists)

Reply:
Sounds good then.